guides/common/modules/con_provisioning-management-with-foreman.adoc

```diff
@@ -0,0 +1,4 @@
+[id="Provisioning-Management-with-{ProjectNameID}_{context}"]
+= Provisioning management with {ProjectName}
+
+With {ProjectName}, you can provision hosts on various compute resources with many provisioning methods from a centralized console.
```

Comment:
"centralized console" > unified interface? There's more than console as in "CLI", but also API, FAM, and Web UI.

Reply:
Applied.